templates/udev_conf.epp

```diff
@@ -0,0 +1,23 @@
+# This file managed by Puppet - DO NOT EDIT
+#
+# The initial syslog(3) priority: "err", "info", "debug" or its
+# numerical equivalent. For runtime debugging, the daemons internal
+# state can be changed with: "udevadm control --log-priority=<value>".
+<% if scope['systemd::udev_log'] -%>
```

Comment:
I would rather pass the parameters explicitly.

Reply:
something like this work?